### ISSUE TITLE
Python kit is not runnable

### ISSUE THREAD

**Issue by KHN190** [synapsepayments/synapse_pay-python#3]
In the first place I found in your [requeter.py](https://github.com/synapsepayments/synapse_pay-python/blob/master/synapse_pay/apibits/requester.py) line 15:

```
requests.packages.urllib3.disable_warnings()
```

where basically I assumed the requests lib refers to this: [requests HTTP for humans](http://docs.python-requests.org/en/latest/). However, `requests` doesn't have module `packages`. I didn't figure out the way it works, since, sure, we have `urllib3`, but it's independent from `requests`. How does this work?

When I removed this line, and tried to create a user, 

```
import synapse_pay

synapse_pay.CLIENT_ID = "***"
synapse_pay.CLIENT_SECRET = "***"

client = synapse_pay.User.create({
    "email" : "***",
    "fullname" : "***",
    "ip_address" : "***",
    "phonenumber" : "***",
    "dp" : "",
    "password" : "***",
})
```

The lib tried to call requests with unmatched args:

```
...
File "/usr/lib/python2.7/dist-packages/urllib3/util.py", line 147, in _validate_timeout "int or float." % (name, value)) 
ValueError: Timeout value connect was (30, 60), but it must be an int or float.
```

I stopped here because I think this library might not work the way I presumed.

**Comment by joncalhoun** [comment]
Hi @KHN190 

What are you trying to do, and are you having an issue with the library itself, or are you asking so you can understand the lib better?

**Comment by KHN190** [comment]
@joncalhoun I don't think the Python lib is runnable. I might be wrong. (But see above for example.)

**Comment by joncalhoun** [comment]
What version of python are you running, and can you show me the exact steps you took to install the library and reproduce the error?

The library is working on my end, so info on how to reproduce your error would help a lot. Thx!

**Comment by KHN190** [comment]
@joncalhoun Python 2.7.
- `requests` installed by `pip install requests`.
- I installed the lib with `pip install synapsepay` (it found and reinstalled `requests`, which is the same version, 2.2.1)

And output:

```
>>> import synapsepay
Traceback (most recent call last):
File "<stdin>", line 1, in <module>
File "/usr/local/lib/python2.7/dist-packages/synapsepay/__init__.py", line 8, in <module> from .client import Client
File "/usr/local/lib/python2.7/dist-packages/synapsepay/client.py", line 1, in <module>
from .apibits import APIClient, APIMethod
File "/usr/local/lib/python2.7/dist-packages/synapsepay/apibits/__init__.py", line 8, in <module>
from .requester import Requester
File "/usr/local/lib/python2.7/dist-packages/synapsepay/apibits/requester.py", line 15, in <module>
    requests.packages.urllib3.disable_warnings()
AttributeError: 'module' object has no attribute 'packages'
>>> 
```

`requests` doesn't have attribute `packages`. You must developed the lib with old version `requests` which had functions deprecated since a long time ago.

**Comment by KHN190** [comment]
I tested with Python 3.4:
- `pip3 install synapsepay`

Output:

```
>>> import synapsepay
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.4/dist-packages/synapsepay/__init__.py", line 8, in <module>
from .client import Client
  File "/usr/local/lib/python3.4/dist-packages/synapsepay/client.py", line 1, in <module>
from .apibits import APIClient, APIMethod
  File "/usr/local/lib/python3.4/dist-packages/synapsepay/apibits/__init__.py", line 8, in <module>
from .requester import Requester
  File "/usr/local/lib/python3.4/dist-packages/synapsepay/apibits/requester.py", line 15, in <module>
requests.packages.urllib3.disable_warnings()
AttributeError: 'module' object has no attribute 'packages'
>>> 
```

So in any way in my end the library doesn't run.

**Comment by joncalhoun** [comment]
Could you try installing `requests` version `2.7.0`? It looks like we need to update the setup to enforce a more recent version of requests than what you were using.

**Comment by joncalhoun** [comment]
Setup is fixed in #5, and this will be pushed to pypi before long.
